packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -482,6 +490,28 @@ export class Vector2 {
         result.y = array[offset + 1];
     }
 
+	/**
+	 * Gets the angle between 2 vectors
+	 * @param vector0 defines 1st vector
+	 * @param vector1 defines 2nd vector
+	 * @returns the angle between vector0 and vector1
+	 */
+	public static GetAngleBetweenVectors(vector0: Vector2, vector1: Vector2): Number{
+		let diff = vector1.subtract(vector0);
+		return Math.atan(diff.y / diff.x);
+	}
+
+	/**
+	 * Converts a polar coordinate Vector2 to its rectangular coordinates
+	 * @param polar the polar coordinates
+	 * @returns the rectangular coordinates
+	 */
+	public static fromPolar(polar: Polar): Vector2{

```

Comment:
A `fromPolarToRef(result: Vector2)` would also be nice to have, to avoid creating a new `Vector2` object each time.